### PR TITLE
Fix issues found by code inspection

### PR DESCRIPTION
**Pull request by akrejcir**

### CODE REVIEW

File: pkg/validation/path.go [hunk 1]
```diff
@@ -30,9 +30,7 @@ import (
 )
 
 var (
-	ErrInvalidJSONPath  error = fmt.Errorf("Invalid JSONPath")
-	ErrMismatchingTypes error = fmt.Errorf("Mismatching type(s)")
-	ErrWrongTypes       error = fmt.Errorf("Wrong type(s)")
+	ErrInvalidJSONPath  = fmt.Errorf("invalid JSONPath")
```
Comment (omeryahud) [inline pkg/validation/path.go:33]: I assume the rest of the errors were unused?

Comment (akrejcir) [reply pkg/validation/path.go:33]: Yes. For this PR I have just run "Inspect Code" in Goland and fixed some issues it found.

**Pull request merged**